.gitignore

```diff
@@ -0,0 +1 @@
+actian-vector-*
```

Comment:
this will ignore the tgz as per the title but it will ignore all files starting with actian-vector. Not sure if this was deliberate or not based on the commit message.

Reply:
It was deliberate.  There are scenarios in the Docker world where you might really want a tarball moved into the image by "ADD".   So I specifically just ignored the actian-vector files.  That was an ancillary commit anyway; this fix is for the commented out optional password line fix.